Webex Connect Flows/How to manually upgrade from v3.3 to v3.4 workflows.MD

```diff
@@ -0,0 +1,10 @@
+
+# How to manually upgrade from v3.3 to v3.4 workflows
+
+In case you have made custom changes to your inbound flows, you can manually upgrade your existing v3.3 flows to v3.4. This ensures that the custom changes are retained and don't have to be repeated. Follow the below steps for the same:
+
+## For Google Business Messages 
+- Two new flows have been added in [Google Business Messages](v3.4/Template/Media%20Specific%20Workflows/Google%20Business%20Messages) as references to create flows with Google Business Message Channel 
+
+## For all other flows
+- There are no changes made to other flows they have been moved from v3.3 to v3.4 in pristine condition
```

Comment:
We can keep only "There are no changes made to other flows"

Reply:
done